Description: Support running tests on Minitest version 5
Origin: https://github.com/sparklemotion/nokogiri/commit/736896b0c837852fd9074b4c2f3353a237946ff2.patch
Author: Lee Jarvis <ljjarvis@gmail.com>
Reviewed-by: Cédric Boutillier <boutil@debian.org>
Last-Update: 2013-05-13

--- a/test/xslt/test_custom_functions.rb
+++ b/test/xslt/test_custom_functions.rb
@@ -63,6 +63,10 @@
         skip("Pure Java version doesn't support this feature.") if !Nokogiri.uses_libxml?
         foo = Class.new do
           include MiniTest::Assertions
+          # Minitest 5 uses `self.assertions` in `assert()` which is not
+          # defined in the Minitest::Assertions module :-(
+          attr_writer :assertions
+          def assertions; @assertions ||= 0; end
 
           def multiarg *args
             assert_equal ["abc", "xyz"], args
