diff --git a/xmlwriter.c b/xmlwriter.c
index 69541b8..c9eb0e3 100644
--- a/xmlwriter.c
+++ b/xmlwriter.c
@@ -2680,7 +2680,7 @@ xmlTextWriterStartCDATA(xmlTextWriterPtr writer)

     xmlListPushFront(writer->nodes, p);

-    count = xmlOutputBufferWriteString(writer->out, "<![CDATA[");
+    /*count = xmlOutputBufferWriteString(writer->out, "<![CDATA[");*/
     if (count < 0)
         return -1;
     sum += count;
@@ -2718,7 +2718,7 @@ xmlTextWriterEndCDATA(xmlTextWriterPtr writer)
     sum = 0;
     switch (p->state) {
         case XML_TEXTWRITER_CDATA:
-            count = xmlOutputBufferWriteString(writer->out, "]]>");
+            /*count = xmlOutputBufferWriteString(writer->out, "]]>");*/
             if (count < 0)
                 return -1;
             sum += count;
